app/main/appium-method-handler.js

```diff
@@ -58,20 +58,40 @@ export default class AppiumMethodHandler {
     return {variableName, variableType, strategy, selector, elements};
   }
 
-  async executeElementCommand (elementId, methodName, args = [], skipScreenshot = false) {
-    const cachedEl = this.elementCache[elementId];
+  async _execute ({elementId, methodName, args, skipScreenshotAndSource}) {
+    let cachedEl;
+    let res = {};
 
-    // Give the cached element a variable name (el1, el2, el3,...) the first time it's used
-    if (!cachedEl.variableName && cachedEl.variableType === 'string') {
-      cachedEl.variableName = `el${this.elVariableCounter++}`;
+    if (elementId) {
+      // Give the cached element a variable name (el1, el2, el3,...) the first time it's used
+      cachedEl = this.elementCache[elementId];
+      if (!cachedEl.variableName && cachedEl.variableType === 'string') {
```

Comment:
you can use `_.isString`

Reply:
The `variableType` is a hard-coded string that identifies what type the element is. It's used by the RecordedActions component which displays recorded code and it uses the variableType to determine if it should be `el.text("String")` or `el.text(123)` etc...